internal/interfaces.go

```diff
@@ -160,10 +160,21 @@ type EnvKeysHookResultItem struct {
 }
 
 type LuaPluginInfo struct {
-	Name              string `luai:"name"`
-	Author            string `luai:"author"`
-	Version           string `luai:"version"`
-	Description       string `luai:"description"`
-	UpdateUrl         string `luai:"updateUrl"`
-	MinRuntimeVersion string `luai:"minRuntimeVersion"`
+	Name              string   `luai:"name"`
+	Author            string   `luai:"author"`
```

Comment:
I think `contributors` is better than `author`, contributors can be an string array.
and in my view, perhaps I don't care about the plugin author, I would prefer to see the plugin providing a homepage or github repo.

Reply:
Ok, let's remove it.